backend/routers/analysis.py

```
@@ -0,0 +1,49 @@
+
+
+@router.post("/run")
+async def run_analysis():
+    csv_path = uploaded_files.get("current")
+    if not csv_path:
+        raise HTTPException(400, "먼저 CSV 파일을 업로드해주세요.")
+
+    try:
+        rating_threshold = analysis_settings.get("rating_threshold", 3)
+        result = run_full_analysis(csv_path, rating_threshold=rating_threshold)
+        return result
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)